1-exercises/A-this/exercise-1.js

```diff
@@ -15,10 +15,13 @@ let person = {
     name: "Adrian",
     age: 32,
     currentAddress: "London",
-    celebrateBirthday: /* todo */
+    celebrateBirthday: function () {
+        return (this.age += 1);
```

Comment:
this works fine, but you don't need to `return` here as you're just wanting to mutate the `age` property